### DISCUSSION TITLE
软件更新完启动不了

### DISCUSSION THREAD

**Discussion by lyppiao** [ywmoyue/biliuwp-lite#823] [General]
win11系统，昨天还正常使用4.7.2。刚刚更新了4.7.3和4.7.3.401，均启动不了。
首先是安装完成后，没有像以前一样自动打开。然后点击软件图标也没有反应。

**Comment by lyppiao** [comment]
x64的包。目前覆盖回4.7.2，可以使用

**Comment by ywmoyue** [comment]
看下系统的事件查看器中Windows日志中的应用程序里面有没有biliuwp的相关错误日志

**Reply by lyppiao** [reply]
> 看下系统的事件查看器中Windows日志中的应用程序里面有没有biliuwp的相关错误日志

日志显示正常

**Reply by ywmoyue** [reply]
@lyppiao 用脚本方式更新试试

4.7.3只做了一些小的修复，没动过依赖和编译相关内容，按理不应该出现这种问题

**Reply by lyppiao** [reply]
> @lyppiao 用脚本方式更新试试
> 
> 4.7.3只做了一些小的修复，没动过依赖和编译相关内容，按理不应该出现这种问题

刚刚把win11从23h2更到了24h2，再安装4.7.3.401就直接覆盖安装成功了。
